Israel National Newspaper Library.js

```diff
@@ -0,0 +1,144 @@
+{
+	"translatorID": "9d8099b7-1c50-4159-9a67-f9fc1fb3b463",
+	"label": "Israel National Newspaper Library",
+	"creator": "Anonymus",
+	"target": "^https:\\/\\/www\\.nli\\.org\\.il\\/(en|he|ar)\\/newspapers\\/.*",
```

Comment:
Slashes don't need to be escaped outside JS regex literals, and no need for `.*` at the end (this is a substring matcher, it's implied):

```suggestion
	"target": "^https://www\\.nli\\.org\\.il/(en|he|ar)/newspapers/",
```